napi/parser/index.js

```diff
@@ -1,14 +1,15 @@
-import * as bindings from './bindings.cjs';
-export * from './bindings.cjs';
+const bindings = require('./bindings.js');
 
-export function parseSync(...args) {
-  const result = bindings.parseSync(...args);
-  result.program = JSON.parse(result.program);
-  return result;
-}
-
-export async function parseAsync(...args) {
-  const result = await bindings.parseAsync(...args);
-  result.program = JSON.parse(result.program);
-  return result;
-}
+module.exports = {
+  ...bindings,
+  parseSync(...args) {
+    const result = bindings.parseSync(...args);
+    result.program = JSON.parse(result.program);
+    return result;
+  },
+  async parseAsync(...args) {
+    const result = await bindings.parseAsync(...args);
+    result.program = JSON.parse(result.program);
+    return result;
+  },
+};
```

Comment:
So that NodeJS's CJS module lexer can see all the exports and give a nice result when you `import` the package, would be better to export each individually, like NAPI-RS does:

https://github.com/oxc-project/oxc/blob/2de9ef578ecac4d9c99cfca171a0dbdbb4d9a66c/napi/parser/bindings.js#L345-L349

Sorry to be a pain!

Reply:
Yikes, this is why moving to esm would be nice. This is going to be a pain to maintain.